### ISSUE TITLE
Design and create "calls to action" for the 6 text blocks in the landing page

### ISSUE THREAD

**Issue by maboa** [hyperaudio/2020-landing#2]
These can be additional buttons/links or just links directly from the actual text that will take you to a dedicated page for each category.

We should bear accessibility in mind.
